testing/kokoro/system_tests.sh

```diff
@@ -32,10 +32,12 @@ go version
 date
 
 cd "${1:-github/golang-samples}"
+PROJECT_ROOT=$(pwd)
 
 export GO111MODULE=on # Always use modules.
 export GOPROXY=https://proxy.golang.org
 TIMEOUT=60m
+export GOLANG_SAMPLES_E2E_TEST=
```

Comment:
It looks like this is missing an assignment on the right-hand side.

Reply:
well, its explicitly set to an empty string, so that `testutil` considers this value false. But adding an explicit empty string is clearer, so i've done that.